core/domain/exp_services_test.py

```diff
@@ -447,16 +447,10 @@ def mock_delete_docs(doc_ids, index):
         with delete_docs_swap:
             exp_services.delete_exploration(self.owner_id, self.EXP_ID)
 
-    def test_create_new_exploration_error_cases(self):
+    def test_no_errors_are_raised_when_creating_default_exploration(self):
```

Comment:
Should there be a similar test ensuring default explorations do fail strict saving? Or perhaps better yet, that default explorations cannot be published?